### PR TITLE
remove user from team

### PR DESCRIPTION
**Pull request by tejaswinichile**
Issue #316

### CODE REVIEW

**Comment by tejaswinichile** [comment]
@breezyboa _a

File: app/javascript/Admin/Page/Team/List.elm [hunk 1]
```diff
@@ -104,5 +114,22 @@ row teamMember =
...
             []
             [ text <| (teamMember.name ++ " | " ++ teamMember.email) ]
+        , div
+            []
+            [ Html.a
+                [ onClick (DeleteTeamMember teamMember.email)
+                , class "button primary deleteTeamMember"
+                ]
+                [ text "Remove Team Member" ]
+            ]
```
Comment (breezyboa) [inline app/javascript/Admin/Page/Team/List.elm]: Can you keep the UI layout consistent

Similar to this : 
url-list

Comment (tejaswinichile) [reply app/javascript/Admin/Page/Team/List.elm]: @breezyboa We will handle the same in this ticket >> https://github.com/bigbinary/acehelp/issues/366

**Comment by tejaswinichile** [comment]
@breezyboa _a

**Comment by breezyboa** [comment]
@abdul-shajin _a please review rails code

**Pull request merged**